README.md

```diff
@@ -14,7 +14,8 @@ This repo currently offers evaluation metrics for two vision tasks:
     - `EceLossEvaluator`: computes the [ECE loss](https://arxiv.org/pdf/1706.04599.pdf), i.e., the expected calibration error, given the model confidence and true labels for a set of data points. 
 - **Object detection**:
     - `MeanAveragePrecisionEvaluatorForSingleIOU`, `evaluators.MeanAveragePrecisionEvaluatorForMultipleIOUs`: computes the mean average precision (mAP), i.e. mean average precision across different classes, under single or multiple [IoU(s)](https://en.wikipedia.org/wiki/Jaccard_index).
-    
+    - `evaluators.CocoMeanAveragePrecisionEvaluator`: Coco mAP computation for multiple IoUs.
```

Comment:
- Can we add it to vision_evaluation/__init__.py? 
- We can probably remove the MeanAveragePrecisionEvaluatorForSingleIOU and MeanAveragePrecisionEvaluatorForMultipleIOUs from the readme to demote the usage, or move them under CocoMeanAveragePrecisionEvaluator and update the readme to mark it as legacy.
- No need for "evaluators." prefix here. (actually same for MeanAveragePrecisionEvaluatorForMultipleIOUs, not part of this PR)